From scjody@conscoop.ottawa.on.ca Mon Nov 21 13:14:15 2005
Date: Mon, 21 Nov 2005 15:30:23 -0500
From: Jody McIntyre <scjody@steamballoon.com>
To: Pavel Machek <pavel@suse.cz>
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Linus Torvalds <torvalds@osdl.org>, Andrew Morton <akpm@osdl.org>
Subject: Clarify T: field in MAINTAINERS
Message-ID: <20051121203023.GG20781@conscoop.ottawa.on.ca>
Content-Disposition: inline

Pavel Machek points out that for git repos, what we include is not
actually a URL.  It is undesirable to use a URL since git repos can be
accessed in many different ways.

Signed-off-by: Jody McIntyre <scjody@steamballoon.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/MAINTAINERS
+++ gregkh-2.6/MAINTAINERS
@@ -58,7 +58,7 @@ P: Person
 M: Mail patches to
 L: Mailing list that is relevant to this area
 W: Web-page with status/info
-T: SCM tree type and URL.  Type is one of: git, hg, quilt.
+T: SCM tree type and location.  Type is one of: git, hg, quilt.
 S: Status, one of the following:
 
 	Supported:	Someone is actually paid to look after this.
